src/hub/routes/pipeline.py

```diff
@@ -19,4 +19,18 @@ def run(self) -> None:
             elif r == StaticRoutes.FIREFOX_ROUTE:
                 FirefoxRoute(self.data).route()
             else:
-                raise Exception("We do no support " + str(r))
+                raise Exception(f"We do not support " + {r})
```

Comment:
We should create our own Exception based error class, and then raise that.

Reply:
ie src/shared/exceptions.py

Reply:
done, thanks @scottidler